### PR TITLE
Add new health question in census household

### PR DESCRIPTION
**Pull request by MIslam92**
### What is the context of this PR?
Adding new question including piped answer into question & updated functional and integration tests for new question addition

Card: https://trello.com/c/TyvH3nSJ/1501-add-health-question-s

### How to review 
The usual

### CODE REVIEW

File: data/en/census_household.json [hunk 1]
```diff
@@ -1689,6 +1689,41 @@
                 },
                 {
                     "type": "Questionnaire",
+                    "id": "health",
+                    "questions": [{
+                        "id": "health-question",
+                        "title": "How is <em> {{ [answers.first_name[group_instance], answers.last_name[group_instance]] | format_household_name }} </em> health in general?",
```
Comment (jonnyshaw89) [inline data/en/census_household.json]: You have spaces between the `<em>` and `</em>` and the `{{` and `}}` which makes it look odd when rendered

Comment (jonnyshaw89) [reply data/en/census_household.json]: This also need the possessive filter

**Review by jonnyshaw89 — CHANGES_REQUESTED** [review]

File: data/en/census_household.json (hunk 1 above)
Comment (MIslam92) [reply data/en/census_household.json]: opps my bad fixed now

**Comment by jonnyshaw89** [comment]
@MIslam92 this PR contains changes other then what is described in the Trello card. can you attach the other cards if this PR is grouping changes?

**Comment by MIslam92** [comment]
@jonnyshaw89 reverted back now to only my changes

**Comment by jonnyshaw89** [comment]
@MIslam92 all of them?

**Comment by MIslam92** [comment]
@jonnyshaw89 yes allllll of them 🙃

**Review by jonnyshaw89 — APPROVED** [review]

**Review by dwyeradam — APPROVED** [review]
@MIslam92  Squishy Squashy

**Pull request merged**